src/modules/iotjs_module_process.c

```diff
@@ -383,10 +390,12 @@ jerry_value_t InitProcess() {
   // Set iotjs
   SetProcessIotjs(process);
   bool wait_source = false;
+#ifdef JERRY_DEBUGGER
```

Comment:
IMHO, this way would be better:

```c
#ifdef JERRY_DEBUGGER
  bool wait_source = false;
  if (iotjs_environment_config(iotjs_environment_get())->debugger != NULL) {
    wait_source = iotjs_environment_config(iotjs_environment_get())
                      ->debugger->wait_source;
  }

  if (!wait_source) {
    SetProcessArgv(process);
  }
#else
  SetProcessArgv(process);
#endif
```

Reply:
isnt compiler figuring it out itself and strip that unused var ?

Reply:
and this same var is used just after so I prefer to declare it false that duplicating the 2 lines

I assume compiler is smart enough to generate the same binary.